.github/workflows/avm.platform.assign-issue.yml

```diff
@@ -0,0 +1,38 @@
+# Workflow for notifying and assigning issues on creation
+name: Issue Creation - notify and assign team
+
+on:
+  issues:
+    types: [opened]
+
+  # Allows you to run this workflow manually from the Actions tab
```

Comment:
What would be the behavior if triggered manually? Which issues would be targeted?

Reply:
No issue will be targeted. I will remove the manual dispatch